portal/src/main/scala/cromwell/pipeline/datastorage/dto/User.scala

```diff
@@ -1,16 +1,24 @@
 package cromwell.pipeline.datastorage.dto
 
+import play.api.libs.json.{ Json, OFormat }
 import slick.lifted.MappedTo
 
 final case class User(
   userId: UserId,
-  email: String,
-  passwordHash: String,
-  passwordSalt: String,
-  firstName: String,
-  lastName: String,
+  email: String = "",
+  passwordHash: String = "",
+  passwordSalt: String = "",
+  firstName: String = "",
+  lastName: String = "",
```

Comment:
Using an empty string as a placeholder is a terrible idea. As @ilyaonishenko said, only bad guys do things like this.
If you are planning to use a case class, you _must_ provide a valid values for all of its fields. If you don't have these values, you should not use this case class at all. In this case, you may create your own case class, for example.